### ISSUE TITLE
Enhance detail of error messages

### ISSUE THREAD

**Issue by jordanCain** [zowe/zlux#289]
Currently when we encounter a none 2** response from an api request we say what has failed but don't propagate any response message to the user.

We should capture response messages and display them in the Snackbar
#depends on https://github.com/zowe/data-sets/issues/14

**Comment by DivergentEuropeans** [comment]
@jordanCain If this is still an issue, we can use Notification framework to display a Zowe notification if something failed. Or a simple Angular Material Snackbar can be added too, like in Zowe Editor 🌱

**Comment by jordanCain** [comment]
@DivergentEuropeans The explorer apps already hook into the notification framework, this issue is more around improving the content of the notifications that are sent, in.some instances we just use a generic failed message without a reason whereas if available we should use a message returned from the api

**Comment by NakulManchanda** [comment]
@jordanCain is this still the case?

**Comment by jordanCain** [comment]
@NakulManchanda Yes, in many cases across the explorers we use generic hardcoded error messages when an action fails rather than relaying what the api/zosmf tells us
